<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Norwidget</class>
 <widget class="QWidget" name="Norwidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0">
    <widget class="QPushButton" name="norbackbon">
     <property name="maximumSize">
      <size>
       <width>100</width>
       <height>50</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>23</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>X</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="3">
    <widget class="QWidget" name="widget" native="true">
     <widget class="QLabel" name="norobj1">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>80</y>
        <width>91</width>
        <height>71</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="image.qrc">:/Photo/obj1.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="norlabel">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>50</y>
        <width>54</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="image.qrc">:/Photo/character.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="norobj2">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>70</y>
        <width>91</width>
        <height>81</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="image.qrc">:/Photo/obj1.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLCDNumber" name="score">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>30</y>
        <width>100</width>
        <height>85</height>
       </rect>
      </property>
     </widget>
     <zorder>norobj1</zorder>
     <zorder>norobj2</zorder>
     <zorder>norlabel</zorder>
     <zorder>score</zorder>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QPushButton" name="NORBON">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>100</height>
      </size>
     </property>
     <property name="text">
      <string>跳跃！</string>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QLCDNumber" name="lcdNumber">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>50</height>
      </size>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="image.qrc"/>
 </resources>
 <connections/>
</ui>
